SUB3/app.py

```diff
@@ -175,36 +188,64 @@ def show_m1display(position):
 
 
         # Pause button flashing
+        
+        if not frame.paused and not frame.running:
+            print("Illegal state: not frame.paused and not frame.running. Corrected to frame.paused and not frame.running.")
```

Comment:
Are we printing this just incase we ever come up with this issue?

Reply:
Yes